packages/main/src/Link.js

```diff
@@ -158,6 +163,15 @@ class Link extends WebComponent {
 		return LinkRederer;
 	}
 
+
+	onBeforeRendering() {
+		const needsNoReferrer = this.target === "_blank"
+			&& this.href
+			&& this._isCrossOrigin();
```

Comment:
Can we add the "noreferrer" attribute for all anchors with target _blank without checking if
the href is cross origin. If it does not have side effects for same origin anchors, let`s have it.

Reply:
Issue #27 shows a reason:

> Important: Only to cross-origin links, otherwise inner-site statistic might be broken.